packages/v4/patternfly-docs/content/get-started/about.md

```diff
@@ -104,3 +104,8 @@ for further updates pending testing and feedback, then the newly introduced CSS
 as well as relevant examples in the documentation are all labeled as beta.
 
 For more information about beta components, visit [this page](https://github.com/patternfly/patternfly-org/tree/main/beta-component-promotion).
+
+## Requesting new features and enhancements
+PatternFly is built on the needs of our community of stakeholders. To request a new feature or an enhancement to an existing feature, the first step is to open a new discussion topic in the [PatternFly Feature Request discussion forum](https://github.com/orgs/patternfly/discussions/categories/feature-requests). 
+
+The PatternFly team will review and prioritize your issue, taking into account scope and technical constraints. If accepted, your feature request will be placed on the PatternFly feature roadmap for fruther design and development. After this, the PatternFly design team will work with you to create a design proposal and facilitate reviews.
```

Comment:
```suggestion
The PatternFly team will review and prioritize your issue, taking into account scope and technical constraints. If accepted, your feature request will be placed on the [PatternFly feature roadmap](https://github.com/orgs/patternfly/projects/4?fullscreen=true) for further design and development. After this, the PatternFly design team will work with you to create a design proposal and facilitate reviews.
```

Reply:
Purposely did not link this while it's still work in progress.

Reply:
Makes sense! I removed the link and just left the correction of a small typo